lib/tasks/test.rake

```diff
@@ -3,9 +3,13 @@ require_relative './evm_test_helper'
 if defined?(RSpec)
 namespace :test do
   task :initialize do
-    ENV['RAILS_ENV'] ||= "test"
+    if ENV['RAILS_ENV'] && ENV["RAILS_ENV"] != "test"
+      warn "Warning: RAILS_ENV is currently set to '#{ENV["RAILS_ENV"]}'. Forcing to 'test' for this run."
+    end
+    ENV['RAILS_ENV'] = "test"
+
     Rails.env = ENV['RAILS_ENV'] if defined?(Rails)
```

Comment:
should this just be `Rails.env = 'test' if defined?(Rails)`?

Reply:
Yeah, I was thinking the same thing.

That said, I was also trying to determine the best way we could possibly do this in a "block", where it is only valid for the duration of the block.  Right now, it will just set it for every subsequent task, which might be a bad thing.

Still, might be fine for now.